tajo-core/src/main/java/org/apache/tajo/engine/function/datetime/DatePartFromTimestamp.java

```diff
@@ -59,8 +55,9 @@ public DatePartFromTimestamp() {
 
   @Override
   public void init(OverridableConf context, FunctionEval.ParamType [] types) {
-    String timezoneId = context.get(SessionVars.TIMEZONE, TajoConstants.DEFAULT_SYSTEM_TIMEZONE);
-    timezone = TimeZone.getTimeZone(timezoneId);
+    if (!hasTimeZone()) {
```

Comment:
Actually, this behavior looks to be intended. However, it also looks weird. What do you think?